modules/retail/pages/retail-offline.adoc

```diff
@@ -37,9 +37,27 @@ For more about kernel parameters, see xref:salt:formula-pxe.adoc[].
 
 If there is low bandwidth or an unstable connection between the branch server and the terminal, images can be cached in advance and the upgrade performed on the terminals later on.
 
-// This looks incomplete or not detailed enough
+This functionality requires the terminal to have a dedicated service partition. A service partition is a partition mounted as `/srv/saltboot`.
+This partition must be created before the system partition and large enough to store a system image.
+To ensure that terminals will always have such a partition, use the Saltboot formula for terminal hardware type to specify the partition details.
+For more information, see xref:salt:formula-saltboot.adoc[].
+
+When the service partition is set up on the terminal, a system image can be downloaded in advance by applying the `saltboot.cache_image` state:
+
+----
+salt $TERMINALID state.apply saltboot.cache_image
+----
+
+This can be done regularly to ensure that terminals always have an uptodate system image downloaded.
+
+When the terminal is rebooted and an uptodate system image is found in the service partition, the terminal will automatically use this cached image for system redeployment.
```

Comment:
During the review I started to use POS image instead of system image. I think it is then clear what it is talked about.

Reply:
ok, I changed my local copy.